wp-admin/js/widgets/media-image-widget.js

```diff
@@ -147,11 +142,17 @@
 			 * customizer.
 			 */
 			mediaFrameContentView = mediaFrame.views.get( '.media-frame-content' )[0];
-			mediaFrameContentView.model.dfd.done( function() {
-				_.defer( function() { // Next tick.
-					mediaFrameContentView.$el.find( '[data-setting="caption"]:first' ).focus();
+			if ( mediaFrameContentView.model.dfd ) {
+				mediaFrameContentView.model.dfd.done( function() {
+					_.defer( function() { // Next tick.
+						mediaFrameContentView.$el.find( '[data-setting="caption"]:first' ).focus();
+					} );
 				} );
-			} );
+			} else {
+				_.defer( function() {
+					jQuery( '.media-frame-content [data-setting="caption"]:first' ).focus();
+				} );
+			}
```

Comment:
We could also just remove all of this autofocus logic from the widget itself. To me it seems like this should actually be handled in the media library JS itself, right? I mean, right now it seems that the [`this.$el.focus();` call](https://github.com/WordPress/wordpress-develop/blob/4.7.3/src/wp-includes/js/media-views.js#L6805) in `Modal#focus()` should perhaps be improved to automatically autofocus on the first frame's first focusable element?